flink-container/src/test/java/org/apache/flink/container/entrypoint/StandaloneJobClusterConfigurationParserFactoryTest.java

```diff
@@ -47,9 +54,21 @@
  */
 public class StandaloneJobClusterConfigurationParserFactoryTest extends TestLogger {
 
+	@Rule
+	public TemporaryFolder tempFolder = new TemporaryFolder();
+	private File confFile;
+	private String confDirPath;
+
+	@Before
+	public void createEmptyFlinkConfiguration() throws IOException {
+		File confDir = tempFolder.getRoot();
+		confDirPath = confDir.getAbsolutePath();
+		confFile = new File(confDir, GlobalConfiguration.FLINK_CONF_FILENAME);
+		new FileOutputStream(confFile).close();
```

Comment:
I would suggest to replace with `confFile.createNewFile();`

Reply:
Done